### ISSUE TITLE
No Custom Date Ranges in New UI

### ISSUE THREAD

**Issue by mondus** [TelemetryDeck/Viewer#421]
A similar issue has been raised (#334) which was closed with reference to another issue which has a 404. 

Custom date ranges are no longer possible and as such it is no longer possible to get a count of all signals/users (for all time).

**Comment by winsmith** [comment]
Thanks – the ticket in question is https://github.com/TelemetryDeck/web/issues/1475, which is in a private project. I'm leaving this issue open and we'll close both once the new date picker UI is done.

**Comment by mondus** [comment]
As a short term fix is there any know way to obtain the total signals and unique users for a custom duration using TQL?

**Comment by Jeehut** [comment]
@mondus FYI: I'm actively working on resolving this issue by adding a custom date range option to the new date picker. Should be available soon.

In the mean time, you can in fact use TQL to specify ranges for insights, here's the related doc:
https://telemetrydeck.com/docs/tql/time-intervals/
